partials/AlertText.txt

```diff
@@ -0,0 +1 @@
+<a href='https://blog.chocolatey.org/'>Looking to master Chocolatey Packaging? Check the 12 Days of Chocolatey Packaging going on right now!</a>
```

Comment:
```suggestion
<a href='https://blog.chocolatey.org/tags/packaging'>Looking to master Chocolatey Packaging? Check the 12 Days of Chocolatey Packaging going on right now!</a>
```

What if we linked directly to the `packaging` tag on the blog. This way if someone clicks the link and bookmarks it, they can always get back to the posts.